src/components/transactions/message-view/DelegateMessageDetails.vue

```diff
@@ -23,22 +23,21 @@
     </div>
     <div class="tx__content__right">
       <p class="amount">
-        {{ coin.amount | atoms | prettyLong }} {{ coin.denom | viewDenom }}
+        {{ coin.amount | prettyLong }} {{ coin.denom | viewDenom }}
```

Comment:
viewDenom should not be necessary right?

Reply:
I guess not, yes. We will send it all ready and well packaged from the BE